tools/cpp/wrapper/bin/pydir/msvc_tools.py.tpl

```diff
@@ -357,11 +363,27 @@ class ArgParser(object):
       i += num_matched
     self.leftover = unmatched
 
-    # Use the proper runtime flag depending on compilation mode. If the
-    # compilation is happening in debug mode, this flag already exists. If not,
-    # then we must add it.
-    if '/MT' not in self.options and '/MTd' not in self.options:
-      self.AddOpt('/MT')
+    # Select runtime option
+    # Find the last runtime option passed
+    rt = None
+    rt_idx = -1
+    for i, opt in enumerate(reversed(self.options)):
+      if opt in ['/MT', '/MTd', '/MD', '/MDd']:
+        if opt[-1] == 'd':
+          enforce_debug_rt = True
+        rt = opt[:3]
+        rt_idx = len(self.options) - i - 1
+        break
+    rt = rt or '/MD'  # Default to dynamic runtime
```

Comment:
Well I read [here](http://stackoverflow.com/a/14752373/1782792) that it is the default option in Visual Studio and [here](https://stackoverflow.com/questions/14172856/cmake-compile-with-mt-instead-of-md) that it is too in CMake. But honestly I don't think there are unquestionable reasons for one or another...